_config.yml

```diff
@@ -52,6 +52,7 @@ requires_5_1v: "> **Note**: This feature is only available for TinyMCE 5.1 and l
 requires_5_2v: "> **Note**: This feature is only available for TinyMCE 5.2 and later."
 requires_5_3v: "> **Note**: This feature is only available for TinyMCE 5.3 and later."
 requires_5_4v: "> **Note**: This feature is only available for TinyMCE 5.4 and later."
+requires_5_5v: "> **Note**: This feature is only available for TinyMCE 5.5 and later."
```

Comment:
```suggestion
```

Reply:
Added on https://github.com/tinymce/tinymce-docs/pull/1673/files

Reply:
This shouldn't be needed, as it'll detect it's the same change and just merge them.